Updated requirements.txt

swapnilj01 opened the pull request:
Changed a dependency that created issues while installing via pip

swapnilj01 committed:
Updated requirements.txt

Changed a dependency that created issues while installing via pip

gdvalderrama commented:
Dependencies should have a pinned version.
@swapnilj01 If you're going to change it at least pin it to the version that works for you.

swapnilj01 committed:
Updated requirements.txt

Updated psycopg package to support some docker host machines (ex Apple M1 chips)
